src/utils/agentjs-cbor-copy.utils.ts

```diff
@@ -1,25 +1,32 @@
-import type {CallRequest, Expiry} from '@dfinity/agent';
+import {Expiry, JSON_KEY_EXPIRY, type CallRequest} from '@dfinity/agent';
 import {decode} from '@dfinity/cbor';
 import {Principal} from '@dfinity/principal';
 import {base64ToUint8Array} from '@dfinity/utils';
 import type {IcrcBlob} from '../types/blob';
 
+// Expiry doesn't have a fromBigInt static method yet
+const bigIntToExpiry = (val: bigint) => {
```

Comment:
Can you add a TODO for this? Once this PR reviewed and merged we need to extract the function in a utility and provide some specific tests.